project-incubation-exit.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Perhaps this is a good time to import the pages from `toc.hyperledger.org` into GitHub rather than leave these stub redirect files.